### ISSUE TITLE
[Request] Automatic FreeLeech timer is missing

### ISSUE THREAD

**Issue by BitPyrate** [HDInnovations/UNIT3D-Community-Edition#3690]
https://unit3d.dev/dashboard/automatic-torrent-freeleeches/create

it seems logical to have the ability to sed how long the freeleech lasts rather than indefinite if we want.

**Comment by vincejv** [comment]
I'll work on this once my PR for double upload is merged https://github.com/HDInnovations/UNIT3D-Community-Edition/pull/3727
I'll probably enhance it further for FL and 2X timer on a separate PR
